addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -73,8 +154,160 @@ def _stripe_make_request(self, endpoint, payload=None, method='POST', offline=Fa
             raise ValidationError("Stripe: " + _("Could not establish the connection to the API."))
         return response.json()
 
+    def _get_stripe_request_headers(self, endpoint):
+        """Returns the Stripe API call header
+
+        Note: This method is overridden by the internal module responsible for Stripe Connect.
```

Comment:
Good point mentioning the overriding module. :+1: 

Few devs have access to internal and will quickly assume that the method and its param are useless.